entity-framework/core/extensions/index.md

```diff
@@ -360,3 +360,9 @@ Add Entity Framework `IQueryable` support to GraphQL.
 A standard for implementing REST APIs with specifications for discovery, filtering, sorting, projections, navigations, bulk operations, and more.
 
 [GitHub repository](https://github.com/OData) | [NuGet](https://www.nuget.org/packages/Microsoft.OData.Core/)
+
+### IWAPI
+
+IWAPI (Instant Web API) is a scaffolding engine for .NET Core that can automate the generation of DbContext classes, entities, models and creates a working Web API from any SQL Server database.
+
+[Website](https://instantwebapi.com/) | [GitHub repository](https://github.com/ExesoftInc/IWAPI) 
```

Comment:
Remove link to github repository unless you plan to publish code there.

Reply:
Yes, there is code in the GitHub repository. it is code generated from AdventureWorks sample database using IWAPI. It is using .Net Core 3.1 or 5.0

Reply:
The github repository link refers to the source code of the extensions. Not the sample it generated.